src/performance/metrics/PerformanceUtil.cpp

```diff
@@ -33,58 +35,42 @@ void PerformanceUtil::init() {
-    int hostListSize = hostList.size();
-    int counter = 0;
-    std::vector<std::future<long>> intermRes;
-    PlacesToNodeMapper placesToNodeMapper;
+    // Host level
+    double cpuUsage = StatisticCollector::getCpuUsage();
+    Utils::send_job("", "cpu_usage", std::to_string(cpuUsage));
 
-    std::string placeLoadQuery =
-        "select ip, user, server_port, is_master, is_host_reporter,host_idhost,idplace from place";
-    std::vector<vector<pair<string, string>>> placeList = perfDb->runSelect(placeLoadQuery);
-    std::vector<vector<pair<string, string>>>::iterator placeListIterator;
+    long rxBytes = StatisticCollector::getRXBytes();
+    Utils::send_job("", "rx_bytes", std::to_string(rxBytes));
 
-    for (placeListIterator = placeList.begin(); placeListIterator != placeList.end(); ++placeListIterator) {
-        vector<pair<string, string>> place = *placeListIterator;
-        std::string host;
-        std::string requestResourceAllocation = "false";
-
-        std::string ip = place.at(0).second;
-        std::string user = place.at(1).second;
-        std::string serverPort = place.at(2).second;
-        std::string isMaster = place.at(3).second;
-        std::string isHostReporter = place.at(4).second;
-        std::string hostId = place.at(5).second;
-        std::string placeId = place.at(6).second;
+    long txBytes = StatisticCollector::getTXBytes();
+    Utils::send_job("", "tx_bytes", std::to_string(txBytes));
 
-        if (ip.find("localhost") != std::string::npos) {
-            host = "localhost";
-        } else {
-            host = user + "@" + ip;
-        }
+    long totalMemoryUsage = StatisticCollector::getTotalMemoryUsage();
+    Utils::send_job("", "total_memory", std::to_string(totalMemoryUsage));
 
-        if (isHostReporter.find("true") != std::string::npos) {
-            std::string hostSearch = "select total_cpu_cores,total_memory from host where idhost='" + hostId + "'";
-            std::vector<vector<pair<string, string>>> hostAllocationList = perfDb->runSelect(hostSearch);
+    long usedSwapSpace = StatisticCollector::getUsedSwapSpace();
+    Utils::send_job("", "used_swap_space", std::to_string(usedSwapSpace));
 
-            vector<pair<string, string>> firstHostAllocation = hostAllocationList.at(0);
+    double currentLoadAverage = StatisticCollector::getLoadAverage();
+    Utils::send_job("", "load_average", std::to_string(currentLoadAverage));
 
-            std::string totalCPUCores = firstHostAllocation.at(0).second;
-            std::string totalMemory = firstHostAllocation.at(1).second;
+    /*
+    long totalSwapSpace = StatisticCollector::getTotalSwapSpace();
+    Utils::send_job("", "total_swap_space", std::to_string(totalSwapSpace));
+    */
 
-            if (totalCPUCores.empty() || totalMemory.empty()) {
-                requestResourceAllocation = "true";
-            }
-        }
+    // Per process
+    /*
+    long memoryUsage = StatisticCollector::getMemoryUsageByProcess();
```

Comment:
Remove the commented lines here if it makes no sense.

Reply:
Removed

Reply:
Ok